libs/ibm/langchain_ibm/chat_models.py

```diff
@@ -47,22 +50,25 @@
     ToolCall,
     ToolMessage,
     ToolMessageChunk,
-    convert_to_messages,
```

Comment:
It seems that LangSmithParams should be rather imported from `langchain_core.language_models.base`

Reply:
removed